routemaster.rb

```diff
@@ -26,14 +26,14 @@ def self.config
   def self.batch_queue
     @_batch_queue ||= begin
       require 'routemaster/models/queue'
-      Models::Queue.new(name: 'main')
+      Models::Queue['main']
```

Comment:
If these are a fixed set the might be nicer to be able to write `Models::Queue.main` or `Models::Queue::Main` instead, to make it clearer that these are fixed and it's not an arbitrary dictionary?